metriq_gym/benchmarks/lr_qaoa.py

```diff
@@ -331,10 +331,10 @@ def dispatch_handler(self, device: QuantumDevice) -> LinearRampQAOAData:
 
         random.seed(seed)  # set seed for reproducibility
         if device.id == "aer_simulator" and graph_type == "NL":
-            graph_device = rx.generators.directed_star_graph(num_qubits)
+            graph_device = rx.generators.star_graph(num_qubits)
         else:
             graph_device = connectivity_graph(device)
-        edges_device = list({tuple(sorted(edge)) for edge in graph_device.edge_list()})
+        edges_device = list({(u, v) for u, v in graph_device.edge_list()})
```

Comment:
Isn't this the same as `graph_device.edge_list()` itself? Or  `list(graph_device.edge_list())`

```
>>> import rustworkx as rx
>>> g = rx.generators.star_graph(5)
>>> g.edge_list()
EdgeList[(0, 1), (0, 2), (0, 3), (0, 4)]
>>> e = list({(u, v) for u, v in g.edge_list()})
>>> e
[(0, 1), (0, 2), (0, 3), (0, 4)]
>>> list(g.edge_list())
[(0, 1), (0, 2), (0, 3), (0, 4)]
```

Reply:
Yes, I changed it.